utilities/tasks/appendWobjectFields/wobjectsOperations.js

```diff
@@ -0,0 +1,45 @@
+const { WObject } = require( '../../../database' ).models;
+const getComments = require( './getComments' );
+const appendObjectParser = require( '../../../parsers/appendObjectParser' );
+const fs = require( 'fs' );
+
+const getWobjects = async () => {
+    try{
+        const result = await WObject.aggregate( [
+            { $match: { fields: { $size: 0 } } },
+            { $project: { author: 1, author_permlink: 1, _id: 0, default_name: 1 } }
+        ] );
+        return { result };
+    }catch( error ) {
+        return { error };
+    }
+};
+
+const appendWobjectFields = async ( wobject ) => {
+    const { result: comments, err } = await getComments( wobject.author, wobject.author_permlink );
+    if ( err ) console.error( err );
+    if ( !comments || !comments.length ) {
+        console.log( `wobject, author: ${wobject.author}, permlink: ${wobject.author_permlink} has no comments with appends` );
+        return true ;
```

Comment:
illogical value of variable, maybe need to return false if wobject has no any comments with fields ???